src/App.js

```diff
@@ -1,19 +1,33 @@
-import React from 'react'
+import React, { useState } from 'react'
 import Card from './components/Card'
 import styled from 'styled-components/macro'
+import Input from './components/Input'
 
 function App({ data }) {
+  const [cardData, setCardData] = useState(data)
+
+  function handleCreateCard({ text, author, id }) {
+    const newData = [...cardData, { id: id, text: text, author: author }]
+
+    setCardData(newData)
+    console.log(newData)
+  }
```

Comment:
You can do it like this, but I think it makes more sense to just have "text" as a parameter in the function. 
Right now, you create an object in the 'Input.js' which basically is the same as in line 10 in which you recreate it.